addons/event/models/event_event.py

```diff
@@ -215,6 +219,8 @@ def _default_event_mail_ids(self):
         compute_sudo=True)
     country_id = fields.Many2one(
         'res.country', 'Country', related='address_id.country_id', readonly=False, store=True)
+    lang_id = fields.Selection(_lang_get, string='Language',
```

Comment:
`lang_id` looks like a many2one, but it's a selection
can you call it `lang` ?

Reply:
@std-odoo Good catch! Done :)